test/integration/clitest/test_render.py

```diff
@@ -416,3 +419,12 @@ def test_set_inactive(self, tmpdir):
         self.args += ["set", self.idonly, str(rdfile)]
         self.cli.invoke(self.args, strict=True)
         self.assert_target_rdef(self.idonly, rd)
+
+    def test_thumb_nordef(self):
```

Comment:
do you mean ``nodef``?

Reply:
I meant `norenderingdef`

Reply:
in that case, maybe rename to ``test_thumb_norenderingdf``, I assume it was a typo